src/SqlPersistence/Timeout/TimeoutCommandBuilder.cs

```diff
@@ -18,6 +18,8 @@ public static TimeoutCommands Build(SqlVariant sqlVariant, string tablePrefix, s
                     return BuildMySqlCommands($"`{tablePrefix}TimeoutData`");
                 case SqlVariant.MsSqlServer:
                     return BuildSqlServerCommands($"[{schema}].[{tablePrefix}TimeoutData]");
+                case SqlVariant.Oracle:
+                    return BuildOracleCommands($"{tablePrefix}TO");
```

Comment:
Time Outs? Was trying to use specific acronyms without resorting to D for Data in all cases, but...

* Time Outs = TO
* Subscription Storage = SS
* Out Box = OB .... hmmmm, OB usually means something else to people